src/App.tsx

```diff
@@ -1,14 +1,45 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+
+const initialTodo: Todo = {
+  userId: 0,
+  id: 0,
+  title: '0',
+  completed: false,
+};
+
+let allTodos: Todo[] = [];
```

Comment:
Using a global variable `allTodos` can lead to potential side effects and makes the code harder to maintain. Consider managing this state within the component using React's state management hooks.